test/testdrive/rename.td

```diff
@@ -186,10 +186,11 @@ View                                "Create View"
 ------------------------------------------------------------------------------------------------------------------------------------------------
 materialize.public.dependent_view   "CREATE VIEW \"materialize\".\"public\".\"dependent_view\" AS SELECT * FROM \"materialize\".\"public\".\"renamed_mz_view\""
 
+$ set-regex match=u\d+ replacement=UID
```

Comment:
Minor, but maybe toss this at the top of the file so that it applies if someone adds another UID-dependent view higher up?